docs/user-guide/Testing.md

```diff
@@ -71,6 +71,7 @@ Each request is represented by a hash of its definition.
         "status_code": "400",
         "status_text": "BAD REQUEST",
         "error_message": "{\n    \"errors\": {\n        \"id\": \"'5882' is not of type 'integer'\"\n    },\n    \"message\": \"Input payload validation failed\"\n}\n",
+        "todo marina: update doc": ""
```

Comment:
Don't you want to finish this doc update as part of this PR?

Reply:
I mean: please eliminate this todo, or update the doc further if need be.

Reply:
Oops, looks like this did not get committed.  Fixing.

---
In reply to: [604565291](https://github.com/microsoft/restler-fuzzer/pull/170#discussion_r604565291) [](ancestors = 604565291)